src/components/calendar/ClassModal.js

```diff
@@ -26,6 +28,21 @@ function ClassModal({ showDialog, toggleDialog, classes, section, removeSection
         <Typography variant="h5">
           {`${section.subjectId} ${section.courseId} Section ${section.sectionNumber}`}
         </Typography>
+        <Typography>
+          {`${section.schedules.map(getFormattedClassSchedule)}`}
```

Comment:
Using the map is correct - but we want each item in the array to return its own `Typography` component instead of just mapping the schedules array to strings like this. So inside the map you'll have to use an arrow function that returns some JSX. Does that make sense? This applies to all of the following maps in this file as well. See if you can figure it out on your own, but if not you can take a look at `Cart.js` to see an example of a map returning some JSX.